_topic_map.yml

```diff
@@ -2283,6 +2283,8 @@ Topics:
   Topics:
   - Name: How Knative Eventing works
     File: serverless-knative-eventing
+  - Name: Getting started with Knative Eventing
```

Comment:
I find this chapter hard to find. If you start reading from previous one _"How Knative Eventing works"_, no link is guiding you to this one. Instead, you are guided to next chapters on brokers, channels, triggers etc.

Also, for me it's a bit confusing. If I would like to get to know what Knative Eventing is I wouldn't probably know should I check out _"How Knative Eventing works"_ or maybe _"Getting started with Knative Eventing"_. Both of them sound like a good starting point. Maybe better would be to join them into single page?

Reply:
Great suggestion

Reply:
Doing some work to fix how we discuss components in this PR: https://github.com/openshift/openshift-docs/pull/23920
I'll rebase on top of this if possible or open new PRs to redo this content